Documentation/ceph-object-store-crd.md

```diff
@@ -97,6 +99,12 @@ The gateway settings correspond to the RGW daemon settings.
 * `resources`: Set resource requests/limits for the Gateway Pod(s), see [Resource Requirements/Limits](ceph-cluster-crd.md#resource-requirementslimits).
 * `priorityClassName`: Set priority class name for the Gateway Pod(s)
 
+## Zone Settings
+
+The [zone](ceph-object-multisite.md) settings allow the object store to join custom create [ceph-object-zone](ceph-object-multisite-crd.md).
```

Comment:
```suggestion
The [zone](ceph-object-multisite.md) settings allow the object store to join custom create [ceph-object-zone](ceph-object-multisite.md).
```

Reply:
I think it would be best to keep the CRD in there because that's the file that describes a zone resource the object-store is joining rather than what multisite is in general.

Reply:
Don't they need to understand multisite if they are going to set the zone?